src/PlusDataCollection/SonixVideo/vtkPlusSonixPortaVideoSource.cxx

```diff
@@ -391,19 +393,7 @@ PlusStatus vtkPlusSonixPortaVideoSource::AddFrameToBuffer(void* param, int id, b
   customFields["MotorRotatedToTransducerTransform"] = motorRotatedToTransducerTransformStr.str();
   customFields["MotorRotatedToTransducerTransformStatus"] = "OK";
 
-  if (this->ImageToTransducerTransformName != NULL && strlen(this->ImageToTransducerTransformName) > 0)
-  {
-    std::ostringstream imageToTransducerTransformStr;
-    double zPixelSpacingMm = (this->CurrentPixelSpacingMm[0] + this->CurrentPixelSpacingMm[1]) / 2.0; // set to non-zero to keep the matrix as a 3D-3D transformation
-    imageToTransducerTransformStr << this->CurrentPixelSpacingMm[0] << " 0 0 " << -1.0 * this->CurrentTransducerOriginPixels[0]*this->CurrentPixelSpacingMm[0];
-    imageToTransducerTransformStr << " 0 " << this->CurrentPixelSpacingMm[1] << " 0 " << -1.0 * this->CurrentTransducerOriginPixels[1]*this->CurrentPixelSpacingMm[1];
-    imageToTransducerTransformStr << " 0 0 " << zPixelSpacingMm << " 0";
-    imageToTransducerTransformStr << " 0 0 0 1";
-    customFields["ImageToTransducerTransform"] = imageToTransducerTransformStr.str();
-    customFields["ImageToTransducerTransformStatus"] = "OK";
-  }
-
-  PlusStatus status = aSource->AddItem(deviceDataPtr, aSource->GetInputImageOrientation(), frameSize, VTK_UNSIGNED_CHAR, 1, US_IMG_BRIGHTNESS, numberOfBytesToSkip, id, UNDEFINED_TIMESTAMP, UNDEFINED_TIMESTAMP, &customFields);
+  PlusStatus status = this->AddVideoItemToVideoSource(*aSource, deviceDataPtr, aSource->GetInputImageOrientation(), frameSize, VTK_UNSIGNED_CHAR, 1, US_IMG_BRIGHTNESS, numberOfBytesToSkip, id, UNDEFINED_TIMESTAMP, UNDEFINED_TIMESTAMP, &customFields);

```

Comment:
Why did you remove ImageToTransducerTransformName? It is an important feature.

Reply:
It is redundant with the same variable in the base class.

Reply:
OK, I see. The content of this pull request looks good.

It would be better to have the commits a bit more focused, or at least have more accurate commit message ("Cleaning up misnamed variable in vtkPlusUsDevice" headline does not describe very well what was done).

Thanks for working on these.